src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title><%= htmlWebpackPlugin.options.title %></title>
+</head>
+<body>
+    <header class="header">
+        <nav class="nav">
+            <h1>Leaderboard</h1>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <div class="container">
+                <div class="scores-section">
+                    <div class="recent-scores">
+                        <div class="score-panel">
+                            <div class="score-panel-header">
+                                <h2>Recent scores</h2>
+                                <button type="submit" class="btn" id="refresh-score">Refresh</button>
+                            </div>
+                            <ul class="scores" id="scores"></ul>
+                        </div>
+                    </div>
+                    <div class="score-form">
+                        <h2 class="score-form-header">Add your score</h2>
+                        <form action="#">
+                            <div class="field">
+                                <input type="text" name="name" id="your-name" placeholder="Your name">
+                            </div>
+                            <div class="field">
+                                <input type="text" name="score" id="your-score" placeholder="Your score">
+                            </div>
+                            <div>
+                                <button type="submit" class="btn" id="add-score">Submit</button>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+</body>
```

Comment:
- [ ] index.html main contain only root element.
- Hint: At this point, the elements for the project should be made with js and the `index.html` should have only the `<div>` where you build your page from js. For example, you could add a div like `<div id="root"></div>` and then, call this `<div>` on index.js and add all elements with js code.
- When you build all this with js on index.js, try to use ES6 modules to write modular JavaScript with proper ES6 syntax.

Reply:
Applied changes successfully :+1: